lib/enhancers/enzyme/setup-react-native.js

```diff
@@ -1,17 +1,23 @@
+/* global __MOXY_JEST_CONFIG_ENZYME_REACT_NATIVE_IGNORE_EXTRA_MESSAGE_PATTERNS__:true */
 /* eslint-env jest */
 
 'use strict';
 
-const getCallerFile = require('get-caller-file');
 const extendEnzymeMatchers = require('./matchers');
 
 const wrapConsoleMethod = (fn) => (message, ...rest) => {
     // Ignores logs coming from `react-dom` and similar, see: https://github.com/enzymejs/enzyme/issues/831#issuecomment-542851439
     // This is needed as we need JSDOM for Enzyme, while `react-native` doesn't really use it
-    const blockPatternCallerFile = /(react-dom.development).js/i;
-    const blockPatternMessage = /(module is not correctly linked)/i;
+    const ignoreMessagePatterns = [
+        '<.* /> is using incorrect casing. Use PascalCase for React components, or lowercase for HTML elements.',
+        'The tag <.*> is unrecognized in this browser. If you meant to render a React component, start its name with an uppercase letter.',
+        'React does not recognize the `.*` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `.*` instead. If you accidentally passed it from a parent component, remove it from the DOM element.',
+        'Received `.*` for a non-boolean attribute `.*`',
+        'Unknown event handler property `.*`. It will be ignored.',
+        ...__MOXY_JEST_CONFIG_ENZYME_REACT_NATIVE_IGNORE_EXTRA_MESSAGE_PATTERNS__,
+    ].join('|');
 
-    if (!blockPatternMessage.test(message) && !blockPatternCallerFile.test(getCallerFile())) {
+    if (!message.match(new RegExp(ignoreMessagePatterns, 'i'))) {
```

Comment:
Use .test as it was before since it’s more performant.

Reply:
✅